sktime/datasets/_data_io.py

```diff
@@ -35,6 +35,7 @@
     "Sebastiaan Koel",
     "Emilia Rose",
     "Tony Bagnall",
+    "Yair Beer",
```

Comment:
please use GitHub IDs (and change the other names to GitHub IDs too if you don't mind)

not a merge blocker

Reply:
Done, I used .all-contributorsrc for the IDs.